### PR TITLE
docs: expand on TTL flags

### PR DESCRIPTION
**Pull request by ericpaulsen**
this PR adds color to the documentation for our various TTL flags, and where they map to in the UI. the flags in question:

- `--default-ttl`
- `--max-ttl`
- `--failure-ttl`
- `--inactivity-ttl`

### CODE REVIEW

**Review by sharkymark — APPROVED** [review]

**Comment by bpmct** [comment]
@deansheather can you review? want to make sure all of these flags are active

**Comment by matifali** [comment]
Hi @ericpaulsen these are auto generated docs and will be overwritten when `make gen` is run. 
You should update the respective.go files in CLI directory

**Review by matifali — DISMISSED** [review]
You need to update the respective go files in CLI directory and then do `make gen` and `make update-golden-files`

File: site/src/api/api.ts
Comment (matifali) [inline site/src/api/api.ts:1]: Hi @ericpaulsen these look unrelated changes. Can you remove them from this PR?

File: site/src/xServices/templateVersion/templateVersionXService.ts
Comment (matifali) [inline site/src/xServices/templateVersion/templateVersionXService.ts:1]: Same comment here. This looks unrelated to PR scope.

**Review by matifali — APPROVED** [review]
LGTM but can you revet changes to type script files?

File: site/src/api/api.ts
Comment (ericpaulsen) [reply site/src/api/api.ts:1]: @matifali `make gen` modified these files. i'm not sure if I should remove these changes.

Comment (matifali) [reply site/src/api/api.ts:1]: It is OK to discard those changes. It could be due to a different version of libraries in ci and your setup.
